Formula/mariadb.rb

```diff
@@ -93,11 +90,6 @@ def install
       # Reported upstream at https://jira.mariadb.org/browse/MDEV-7209 - this fix can be
       # removed once that issue is closed and the fix has been merged into a stable release.
       mv "storage/mroonga/version", "storage/mroonga/version.txt"
```

Comment:
Not blocking for this PR, but I wonder if:

1. We could have a `mroonga` formula and use that here instead (just like what we do for `groonga`)
2. The comments could be referenced somewhere more prominent above so that they're more easily spotted in future version bump PRs

Reply:
Yep, makes sense, will have a look at both cases in a separate PR. Also want to check if rocksdb plugin works on Apple Silicon.